<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphInfo</class>
 <widget class="QDialog" name="graphInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="dfsLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bfsLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>qweqwe</string>
   </property>
  </widget>
  <widget class="QLabel" name="mstLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Distances</string>
   </property>
  </widget>
  <widget class="QLabel" name="mstLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MST:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dfsLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DFS from AT&amp;T Park: </string>
   </property>
  </widget>
  <widget class="QLabel" name="bfsLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BFS from Target Field: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>graphInfo</receiver>
   <slot>onCloseClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCloseClick()</slot>
 </slots>
</ui>
